mobify/platform-scaffold
# Separate external and internal facing readme sections

# Separate external and internal facing readme sections

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-40
 **Linked PRs**: https://github.com/mobify/generator-progressive-web/pull/8 and https://github.com/mobify/progressive-web-sdk/pull/645

## Changes
- Split up readmes into 2 separate files.

## How to test-drive this PR
- Review `DEVELOPING.md` and `README.md` to ensure that `README.md` contains anything a partner would need, and `DEVELOPING.md` contains anything a Mobifyer would need.
- `DEVELOPING.md` will be stripped from the repo when a new project is generated (as per https://github.com/mobify/generator-progressive-web/pull/8 this PR)
